commit 3a307bd5add3df34665d1e5cb1b646af55ba9653
Author: Jan Kratochvil <jan.kratochvil@redhat.com>
Date:   Fri Jun 9 07:27:50 2017 +0200

    libipt: Fix [-Werror=implicit-fallthrough=] with gcc-7.1.1.

diff --git a/libipt/src/pt_block_decoder.c b/libipt/src/pt_block_decoder.c
index 21783a6..91d5bb3 100644
--- a/libipt/src/pt_block_decoder.c
+++ b/libipt/src/pt_block_decoder.c
@@ -2095,6 +2095,9 @@ static int pt_blk_proceed_no_event_cached(struct pt_block_decoder *decoder,
 		}
 
 		/* Fall through to ptbq_decode. */
+#if __GNUC__ >= 7
+		__attribute__ ((fallthrough));
+#endif
 
 	case ptbq_decode: {
 		struct pt_insn_ext iext;
